index.js

```diff
@@ -102,17 +102,21 @@ function report(file, node, threshold, target, results) {
     result /= length;
 
     if (result >= threshold) {
-        if (result >= SURENESS_THRESHOLD_DEFINITELY) {
-            level = 'Definitely';
-        } else if (result >= SURENESS_THRESHOLD_VERY) {
-            level = 'Very';
+        if (result >= CONFIDENCE_THRESHOLD_HIGH) {
+            confidence = 'high';
+        } else if (result >= CONFIDENCE_THRESHOLD_MODERATE) {
+            confidence = 'moderate';
         } else {
-            level = 'Quite';
+            confidence = 'low';
         }
 
-        message = file.warn(level + ' hard to read sentence', node);
+        message = file.warn(
+            'Hard to read sentence, ' + confidence + ' confidence',
+            node,
+            'hard-to-read'
```

Comment:
Could you change `hard-to-read` > `retext-readability`? One less magic string, and it makes more sense to me.